### ISSUE TITLE
Auto bump linked dependencies of monorepo

### ISSUE THREAD

**Issue by phucdph** [release-it/release-it#1035]
Greeting. Thanks for this amazing tool.

I used `release-it` on my mono repo project as an example below:
     .
    ├── ...
    ├── packages                    
     │   ├── package-a          
     │   ├── package-b   

And `package-b` uses `package-a` internally.
Currently. When using `release-it` to the new version with `@release-it-plugins/workspaces`. All the packages will share the same version.

My question is: Is there any way these packages have a different version? Only bump when have any changes to it.
In the example above, this is how it works:

`package-a` on 0.2.0
`package-b` on 0.2.3

I made some changes to `package-a`, so will be bumped to `0.2.1`, and `package-b` will automatically be bumped to `0.2.4`. (like  [changesets](https://github.com/changesets/changesets))

Thanks for your help.

**Comment by webpro** [comment]
Sounds like an issue for the https://github.com/release-it-plugins/workspaces repo, right? It's a separate project not maintained by me :)

**Comment by phucdph** [comment]
> 

Yes. It is definitely an issue of  `[release-it-plugins/workspaces](https://github.com/release-it-plugins/workspaces)`. But following the monorepo [recipe](https://github.com/release-it/release-it/blob/main/docs/recipes/monorepo.md) (not using `release-it-plugins/workspaces` anymore). Is there any way to archive that release behavior?

**Comment by webpro** [comment]
Yes, it's the default behavior when you use that recipe, it just runs the release-it process for each workspace separately

**Comment by webpro** [comment]
It depends. The release-it process runs for each package separately, so you can version them independently. Indeed the main thing for you here, "auto bump linked dependencies" would need to be resolved differently. So it's not something you can just copy as-is indeed.

Perhaps you could defer the `npm publish` step to Changesets? I think it does this?

**Comment by webpro** [comment]
Closing due to inactivity
